onnx/onnx.in.proto

```diff
@@ -433,6 +433,15 @@ message TypeProto {
     optional TypeProto value_type = 2;
   };
 
+  message Opaque {
+   // When missing, the domain is the same as the model's.
+    optional string domain = 1;
+  // The name is optional but significant when provided.
```

Comment:
For example here, indent is not right. I guess clang-format won't check proto files.

Reply:
Will do.